bedrock/firefox/templates/firefox/firstrun/new-firstrun.html

```diff
@@ -0,0 +1,38 @@
+{% extends "bare.html" %}
+
+{% block site_css %}
+  {% stylesheet 'firefox_new_firstrun' %}
+{% endblock %}
+
+{% block site_js %}
+  {% javascript 'common' %}
+  {% javascript 'firefox_new_firstrun' %}
+{% endblock %}
+
+{% block content %}
+  <div>
+    <div id="scene">
+      <div class="fxaccounts-container">
+        <div id="left-divider">
+          <h1 id="title">{{_('Already using Firefox?')}}</h1>
+          <p>{{_('Sign in to your account and we’ll sync the bookmarks, passwords and other great things you’ve saved to Firefox on other devices.')}}</p>
+          <a href="{{ url('firefox.sync') }}">{{_('Learn more about Firefox Accounts')}}</a>
+        </div>
+      <div id="firefox-logo"></div>
+      <div class="fxaccounts" id="fxa-iframe-config" data-host="{{ settings.FXA_IFRAME_SRC }}" data-mozillaonline-host="{{ settings.FXA_IFRAME_SRC_MOZILLAONLINE }}">
+        <iframe id="fxa" scrolling="no" data-src="{{ settings.FXA_IFRAME_SRC }}signin?utm_campaign=fxa-embedded-form&amp;utm_medium=referral&amp;utm_source=firstrun&amp;utm_content=fx-{{ version }}&amp;entrypoint=firstrun&amp;service=sync&amp;context=iframe&amp;style=chromeless&amp;haltAfterSignIn=true"></iframe>
```

Comment:
Hey, for FxA metrics could we please specify a different `entrypoint=` value here?